abstract.tex

```diff
@@ -0,0 +1,14 @@
+\begin{abstract}
+
+The Vera C. Rubin Observatory will revolutionize all areas of astronomy over the next decade with its unique wide-fast-deep multi-color imaging survey, the Legacy Survey of Space and Time (LSST).
+LSST will produce approximately 20TB of raw data per night, which will be automatically processed by the LSST Science Pipelines 
```

Comment:
I think this should be "The LSST will produce" following the use guide: https://project.lsst.org/documents/name-use-guide

Reply:
Yes, updated